modules/ROOT/pages/oauth2-policies-new.adoc

```diff
@@ -3,25 +3,27 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-To secure an API in Anypoint Platform using OAuth 2.0, you may use the following policies and the matching OAuth 2.0 provider:
+To secure an API in Anypoint Platform using OAuth 2.0, you can use the following policies and the matching OAuth 2.0 provider:
 
-* One of the following policies available in Federated organizations configured with specific OAuth 2.0 providers:
+* Any one of the following policies available in federated organizations, configured with specific OAuth 2.0 providers:
 ** OpenAM OAuth Token Enforcement policy
 ** PingFederate OAuth Token Enforcement policy
 ** OpenID Connect Access Token Enforcement Policy
 * Available in all organizations:
-** xref:external-oauth-2.0-token-validation-policy.adoc[Mule OAuth 2.0 access token enforcement policy] and xref:mule-oauth-provider-landing-page.adoc[Mule OAuth 2.0 Provider]
+** xref:external-oauth-2.0-token-validation-policy.adoc[Mule OAuth 2.0 Access Token Enforcement policy] and xref:mule-oauth-provider-landing-page.adoc[Mule OAuth 2.0 Provider].
 
-Any of these token enforcement policies won't allow access to OAuth 2.0 protected resources if credentials from other than Mule client applications created in Anypoint Platform are used.
+None of these token enforcement policies allow access to OAuth 2.0 protected resources if credentials from non-Mule client applications that are created in Anypoint Platform are used.
```

Comment:
Non Mule client application cannot be created in Anypoint Platform. The statement is incorrect.

Reply:
Kim verify. To me other than Mule means non-Mule. And the earlier text reads exactly the same.

Reply:
The current wording is awkward and passive. If you mean...:

These token enforcement policies do not provide you with access to OAuth 2.0 protected resources if you use credentials from non-Mule client applications created in Anypoint Platform.

...then yes, you are saying that it is possible to create non-Mule client apps in Anypoint Platform.

I THINK you want to say that the policies do not grant access to the protected resources if you try to use non-Mule apps.

Reply:
In Anypoint you can create client application that will have an associated client ID and secret. You can call those apps "Mule client applications". Outside Mulesoft, other companies have similar concepts, creating kind of client applications with and associated client ID and secret. The latter credentials, will not be accepted when requesting a token to the Mule OAuth provider, and if a token is issued using them by a third party OAuth provider, the OAuth token enforcement policies will not consider a request using such token as a valid one.

Reply:
Done! Thanks!